packages/@react-spectrum/dialog/test/DialogContainer.test.js

```diff
@@ -27,7 +27,7 @@ describe('DialogContainer', function () {
   });
 
   beforeEach(() => {
-    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
+    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => setTimeout(cb, 0));
```

Comment:
go ahead and change our start section for this file to 
```
  beforeAll(() => {
    jest.useFakeTimers();
  });

  afterEach(() => {
    jest.runAllTimers();
  });
```

I verified locally it'll work and it's once less incorrect implementation of mock request animation frame. Not your fault, we used to have to do that.